instrumentation/spymemcached-2.12/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spymemcached/SpymemcachedNetworkAttributesGetter.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spymemcached;
+
+import io.opentelemetry.instrumentation.api.semconv.network.ServerAttributesGetter;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+import java.util.Collection;
+import javax.annotation.Nullable;
+import net.spy.memcached.MemcachedConnection;
+import net.spy.memcached.MemcachedNode;
+
+final class SpymemcachedNetworkAttributesGetter
+    implements ServerAttributesGetter<SpymemcachedRequest> {
+
+  @Nullable
+  @Override
+  public String getServerAddress(SpymemcachedRequest request) {
+    MemcachedConnection connection = request.getConnection();
+    if (connection != null) {
+      Collection<MemcachedNode> nodes = connection.getLocator().getAll();
+      if (!nodes.isEmpty()) {
+        SocketAddress socketAddress = nodes.iterator().next().getSocketAddress();
```

Comment:
`server.address` should reflect the server where the request was sent to. Here you are picking the first node from a list, could you explain why this is correct.